project/src/js/ui/round/view/button.js

```diff
@@ -298,5 +298,29 @@ export default {
           () => window.plugins.toast.show(i18n('notes'), 'short', 'bottom')
         )} />
     );
+  },
+  returnToTournament: function(ctrl) {
+    function handler() {
+      ctrl.hideActions();
+      m.route('/tournament/' + ctrl.data.tournament);
+    }
+    return (
+      <button key="returnToTournament" config={helper.ontouch(handler)}>
+        <span className="fa fa-trophy" />
+        Return to Tournament
+      </button>
+    );
+  },
+  withdrawFromTournament: function(ctrl) {
```

Comment:
Ok then let's keep it, but use the translated string. We should be careful to not add too much buttons since the popup size is limited though.
So maybe you did the right thing in the first place: share pgn and copy link should be removed on tournament.